pkg/plugins/plugins.go

```diff
@@ -269,7 +269,7 @@ func (pm *PluginManager) scan(pluginDir string, requireSigned bool) error {
 			}
 		}
 
-		reader, err := os.Open(jsonFPath)
+		reader, err := os.Open(filepath.Clean(jsonFPath))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.